starfish/core/imagestack/imagestack.py

```diff
@@ -50,6 +51,7 @@
     Clip,
     Coordinates,
     CoordinateValue,
+    FunctionSource,
```

Comment:
> Meta-comment: I think logging and reproducibility is very important, but wonder if we're shooting ourselves in the foot re: usability by not accepting functions directly. I know last time we discussed we waffled a bit and then decided on the side of logging.

I still think that is the right choice.  Otherwise, starfish doesn't have much of value-add.